pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -350,104 +350,165 @@
-               text-align: left;
-               flex: 1;
-               font-weight: 500;
+             .tr {
+               text-align: right;
+             }
+             .pt1 {
+               padding-top: 0.25rem;
+             }
+             .pb2 {
+               padding-bottom: 0.5rem;
+             }
+             .pr1 {
+               padding-right: 0.25rem;
+             }
+             .pr2 {
+               padding-right: .5rem;
+             }
+             .dn {
+               display: none;
+             }
+             #main {
+               width: 100%;
+               height: 100%;
              }
              #inner {
-               width: 75%;
-               margin: 0 auto;
-               display: flex;
-               flex-direction: column;
+               position: fixed;
+               top: 50%;
+               left: 50%;
+               transform: translate(-50%, -50%);
+             }
+             @media all and (min-width: 34.375rem) {
+               .tc-ns {
+                 text-align: center;
+               }
+               .pr0-ns {
+                 padding-right: 0;
+               }
+               .dib-ns {
+                 display: inline-block;
+               }
              }
              '''
     ==
     ;body
       ;div#main
         ;div#inner
-          ;h1#topborder:"Welcome"
-          ;h1#ship-name:"{(scow %p our)}"
-          ;h2:"Get passcode by entering +code at the dojo or from Bridge"
+          ;p:"Urbit ID"
+          ;input(value "{(scow %p our)}", disabled "true", class "mono");
+          ;p:"Access Key"
+          ;p.f9.gray2.pt1
+            ; Get key from Bridge, or
+            ;span.mono.pr1:"+code"
+            ; in dojo
+          ==
           ;form(action "/~/login", method "post", enctype "application/x-www-form-urlencoded")
-            ;input(type "password", name "password", placeholder "passcode", autofocus "true");
+            ;input
+              =type  "password"
+              =name  "password"
+              =placeholder  "~sampel-ticlyt-migfun-falmel"
+              =class  "mono"
+              =autofocus  "true";
             ;input(type "hidden", name "redirect", value redirect-str);
-            ;button(type "submit"):"→ Authenticate"
+            ;button(type "submit"):"Continue"
+          ==
+        ==
+        ;footer.absolute.w-100
+          ;div.relative.w-100.tr.tc-ns
+            ;span(class "absolute", style "left: 8px; bottom: 8px;")
+            ; OS 1
+            ;span(class "gray2", style "margin-left: 4px;"): v0.0.1
+            ==
+            ;p.pr2.pr0-ns.pb2
+              ;a(href "https://bridge.urbit.org", target "_blank")
+                ;span.dn.dib-ns.pr1:"Open"
+                ; Bridge ↗
+              ==
+              ;a
+                =href  "https://urbit.live"
+                =style  "margin-left: 8px; color: #2aa779;"
+                =target  "_blank"
+                ; Purchase
```

Comment:
This should not be here. Please remove this.

Reply:
Tagging @urcades

Reply:
Oh yes — this was meant to point to a page in our docs about purchasing planets, rather than a direct link to some market that we don't own.

I think instead, we can link to this page:

https://urbit.org/using/install/

and anchor link to the "Getting an Urbit ID" subheader.

Reply:
Mk, on it. I recall some oral discussion about "leads you to Urbit Live or whatever" but nothing specific ...

Reply:
Yeah I actually think this would be a valid test, if we did A/B/whatever testing — obviously optimizing for the option that gets more people into Urbit is the preferred route, but I have no idea what that might be atm.

My hunch is that this is a decent route to provide for teaching more people about what obtaining an Urbit ID is, but idk — would love to get more data that supports a more firm position.

Reply:
cc @vvisigoth : Figure you might be interested in this train of thought ^